src/edu/csus/ecs/pc2/core/execute/ExecuteTimerFrame.java

```diff
@@ -0,0 +1,237 @@
+// Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+/**
+ * ExecuteTimerFrame
+ * This frame can be used to display status of executing a run.  It implements the IExecutableMonitor
+ * interface used by the Executable class.
+ */
+package edu.csus.ecs.pc2.core.execute;
+
+import java.awt.BorderLayout;
+import java.awt.Color;
+
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+
+/**
+ * @author John Buck
+ *
+ */
+public class ExecuteTimerFrame extends JFrame implements IExecutableMonitor {
+    private JLabel ivjExecuteTimerLabel = null;
+
+    private JPanel ivjJFrameContentPane = null;
+
+    private JLabel ivjTimerCount = null;
+
+    private JButton ivjbtnTerminate = null;
+
+    private IExecuteFrameNotify iNotifyClient = null;
+    
+    public ExecuteTimerFrame()
+    {
+        super();
+        initialize();
+    }
+    
+    /**
+     * Return the ExecuteTimerFrame property value.
+     *
+     * @return javax.swing.JFrame
+     */
+    private void initialize() {
```

Comment:
The javadoc comment above this method doesn't make sense.  It says that the method returns a property value, but the method return type is `void`.

Reply:
Agreed.  Will fix.

Reply:
Fixed and pushed.